packages/web-components/src/components/va-table/va-table-inner/va-table-inner.scss

```diff
@@ -1,3 +1,22 @@
+@forward 'settings';
+@use 'usa-table/src/styles/usa-table';
+@import '~@department-of-veterans-affairs/css-library/dist/tokens/scss/variables.scss';
+
+
+:host([uswds='true']) table, 
+:host([uswds='true']) caption {
+  font-size: 1.06 * $v3-font-base-size;
```

Comment:
Is this font size adjustment to avoid having to update the `formation-overides.scss` file?

Reply:
@jamigibbs - I should have added to the override file - fixed.